package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@subql/indexer-coordinator",
-  "version": "1.1.5-2",
+  "version": "1.1.5-3",
```

Comment:
manual bump is unnecessary

Reply:
Usually don't need to do it, just need to build a version for testing on dev before the PR merged